packages/manager/apps/dedicated/client/app/dedicated/server/server.constants.js

```diff
@@ -3,13 +3,28 @@ export const ELIGIBLE_FOR_UPGRADE = {
   SUBSIDIARIES: ['GB'],
 };
 
+export const NO_AUTORENEW_COUNTRIES = [
```

Comment:
Not yours but we have like at least 2-3 times this constant or its opposite with each time different values (e.g: https://github.com/ovh/manager/blob/master/packages/manager/modules/models/user/user.constants.js#L1). We should think of using a single source of truth

Reply:
Yes it was the point of a PR I did a year ago and that never got merged 😞